### PR TITLE
BU-2: Allow incrementing integer values in cache

### PR DESCRIPTION
**Pull request by paramsingh**
LB uses this feature a lot to update listen counts of users, so if we want to use brainzutils in LB, this feature is necessary.

In order to implement this, I had to add an option in get and set
that allowed not encoding and decoding the values with msgpack
because msgpack encoded values cannot be incremented.

### CODE REVIEW

**Comment by paramsingh** [comment]
@alastair @mayhem : requesting review here. Also, it'd be great if I was a collaborator on this repo so that I could create a new release when this gets merged.

File: test/Dockerfile.py3 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM metabrainz/python:3.5
+FROM metabrainz/python:3.6
```
Comment (mayhem) [inline test/Dockerfile.py3]: Does this change have possible downstream ramifications that need to be considered?

Comment (alastair) [reply test/Dockerfile.py3]: I guess this is the docker file for running tests on travis? As long as LB, CB, MeB web are running on python 3.6 this is not an issue.

Comment (paramsingh) [reply test/Dockerfile.py3]: Yep, this is only for tests and all of LB, CB and MeB run on the `metabrainz/python3.6` image.

Comment (mayhem) [reply test/Dockerfile.py3]: ok, great.

**Pull request merged**